radio/src/translations.cpp

```diff
@@ -1,714 +1,725 @@
-/*

```

Comment:
Yeah, I am reworking the patch to fix all the issues and will only add the lines that are changed.